src/components/Editor/SearchBar.js

```diff
@@ -473,9 +473,10 @@ const SearchBar = React.createClass({
 
   // Renderers
   buildSummaryMsg() {
-    if (this.state.symbolSearchEnabled) {
+    if (this.state.symbolSearchResults.length > 0) {
```

Comment:
I think you still want to check if symbol search is enabled here in addition to this condition.